tools/fiwalk/src/fiwalk_main.cpp

```diff
@@ -106,6 +106,8 @@ void usage(fiwalk &o)
     printf("Ways to make this program run slower:\n");
     printf("    -M = Report MD5 for each file (default on)\n");
     printf("    -1 = Report SHA1 for each file (default on)\n");
+    printf("    -2 = Report SHA256 for each file (default off)\n");
+    printf("    -5 = Report SHA512 for each file (default off)\n");
     printf("    -S nnnn = Perform sector hashes every nnnn bytes\n");
```

Comment:
A cleaner, more extensible approach would have been to modify this to take a name, rather than more single-character options...

Reply:
I like the idea of `--hash md5,sha512` ... That's certainly more extensible from a plug-in idea. It probably simplifies the code. But it's a change from what you have here.